<launch>
	<node  pkg="objects_of_daily_use_finder" type="odu_finder" name="odu_finder" output="screen">
		<param  name="command" type="string" value="/load" />
		<param  name="database_location" type="string" value="$(find objects_of_daily_use_finder)/database/germandeli_big/" />
		<param  name="images_folder" type="string" value="$(find objects_of_daily_use_finder)/data_big/" />
		<param  name="images_for_visualization_directory" type="string" value="$(find objects_of_daily_use_finder)/data_big/" />

		<param  name="image_topic" type="string" value="/prosilica/image_raw" />
		<!--<param  name="image_topic" type="string" value="/narrow_stereo/left/image_rect" />-->
		<param  name="enable_visualization" type="int" value="1" />

                <param  name="template_topic" type="string" value="/odu_finder/TemplateName" />



		<param  name="votes_count" type="int" value="10" /> 

		<param  name="sliding_window_size" type="int" value="10" /> 

		<param  name="tree_k" type="int" value="7" /> 
		<param  name="tree_levels" type="int" value="7" /> 

		<param  name="unknown_object_threshold" type="double" value="0.9" />

		<param  name="enable_clustering" type="int" value="1" />
		<param  name="enable_incremental_learning" type="int" value="0" />
	</node>
</launch>
